README.adoc

```diff
@@ -26,27 +26,21 @@ You’ll explore how to check the health of a microservice with MicroProfile Hea
 == What you'll learn
 
 You will learn how to use the MicroProfile Health to report the health of
-a microservice. A health check allows your microservice to participate in the decision-making process about its health.
+a microservice. A health check allows your microservice to participate in the decision-making process about
+its health.
 
-The MicroProfile Health provides an endpoint that includes the health check results as a
-JSON object. This HTTP request endpoint returns a status code of `200` if the application outcome
-is `UP`. Otherwise, the endpoint returns a status code of `503` if the application outcome is `DOWN`.
+MicroProfile Health allows services to report their health and it publishes overall health status to a defined
+endpoint.  A service reports `UP` if it is available and reports `DOWN` if it is unavailable.  MicroProfile Health reports
+individual service status at the endpoint and indicates the overall status as `UP` if all the services are `UP`.  A service
+orchestrator can then use the health statuses to make decisions.
 
-You will use the health check to assess the state of a service in an application. The state of a service
-is either `UP` or `DOWN`. When the state is `UP`, the service is available.
-When the state is `DOWN`, the service is either unavailable or in maintenance.
-The overall outcome of the application is `UP` only when the checks of all the services are `UP`.
-A service orchestrator can use a `DOWN` result to determine when to replace an instance of a service.
+A service checks its own health by performing necessary self-checks and then report its overall status via
+implementing the API provided by MicroProfile Health.  A self-check can be a check on anything that is
+needed by the service, such as a dependency, a successful connection to an endpoint, a system property, a database, or
+availability of required resources.
```

Comment:
A service checks its own health by performing necessary self-checks and then report[s] its overall status [by] implementing the API provided by MicroProfile Health.  A self-check can be a check on anything that [the service needs], such as a dependency, a successful connection to an endpoint, a system property, a database, or [the] availability of required resources.

Note: We can't use Latin expressions, such as "via."

Reply:
If you want to be very "proper," we can put items in the list in alphabetical order (the availability of require resources, a database, a dependency, a successful connection to an endpoint, or a system property), but I'll leave that up to you. :)

Reply:
Another method of list organization is placing longer items in the list at the end of the list.

Reply:
@evelinec what do u think, should we list them ?